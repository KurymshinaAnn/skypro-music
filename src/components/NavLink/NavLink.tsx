import classNames from "classnames";
import styles from "./NavLink.module.css";

// type FilterButtonType = {
// };

export default function NavLink () {
  return }
