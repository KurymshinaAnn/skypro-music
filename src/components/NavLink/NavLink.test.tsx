import renderer from "react-test-renderer";
import NavLink from "./NavLink";

it("Renders NavLink with different names", () => {
  const navlink = renderer.create(
    <NavLink href="https://google.com">test</NavLink>
  );
  let tree = navlink.toJSON();
  expect(tree).toMatchSnapshot();
});
