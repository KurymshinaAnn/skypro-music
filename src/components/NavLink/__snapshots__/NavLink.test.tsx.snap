// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders NavLink with different names 1`] = `
<li
  className="menuItem"
>
  <a
    className="menuLink"
    href="https://google.com"
  >
    test
  </a>
</li>
`;
